src/webhook/validation/edgeconnect/validation.go

```diff
@@ -21,26 +21,21 @@ type edgeconnectValidator struct {
 	cfg       *rest.Config
 }
 
-func newEdgeConnectValidator(apiReader client.Reader, cfg *rest.Config) admission.Handler {
+func newEdgeConnectValidator(apiReader client.Reader, cfg *rest.Config, clt client.Client) admission.Handler {
```

Comment:
```suggestion
func newEdgeConnectValidator(clt client.Client, apiReader client.Reader, cfg *rest.Config) admission.Handler {
```

https://github.com/Dynatrace/dynatrace-operator/blob/main/doc/coding-style-guide.md#function-parameter-and-return-value-order